convert-to-file-url/convertToFileUrl.js

```diff
@@ -1,6 +1,12 @@
+import path from 'path';
 
-function convertToFileUrl(...args) {
-  return args;
+
+function convertToFileUrl(filePath) {
+  const p = path.join(filePath);
```

Comment:
Perhaps using a more descriptive name for the variable `p` could make the code more readable. What do you feel?

Reply:
@radhey1851 @jaspreet57  Perhaps a name like `fullPath` or `constructedPath`?